community/main.py

```diff
@@ -206,50 +205,225 @@ def update(self, delta_time):
 				self.media_player.queue(random.choice(self.music))
 				self.media_player.play()
 
-		if not self.mouse_captured:
+		if not self.window.mouse_captured:
 			self.player.input = [0, 0, 0]
 
 		self.joystick_controller.update_controller()
 		self.player.update(delta_time)
 
 		self.world.tick(delta_time)
 
+	def update_ui(self):
+		super().update_ui()
+		if self.show_f3:
+			self.update_f3()
+
 	def on_draw(self):
+		super().on_draw()
 		gl.glEnable(gl.GL_DEPTH_TEST)
 		self.shader.use()
 		self.player.update_matrices()
 
-		while len(self.fences) > self.options.MAX_CPU_AHEAD_FRAMES:
-			fence = self.fences.popleft()
+		while len(self.window.fences) > self.window.options.MAX_CPU_AHEAD_FRAMES:
+			fence = self.window.fences.popleft()
 			gl.glClientWaitSync(fence, gl.GL_SYNC_FLUSH_COMMANDS_BIT, 2147483647)
 			gl.glDeleteSync(fence)
 
-		self.clear()
+		self.window.clear()
 		self.world.prepare_rendering()
 		self.world.draw()
 
-		# Draw the F3 Debug screen
-		if self.show_f3:
-			self.f3.draw()
-
 		# CPU - GPU Sync
-		if not self.options.SMOOTH_FPS:
+		if not self.window.options.SMOOTH_FPS:
 			# self.fences.append(gl.glFenceSync(gl.GL_SYNC_GPU_COMMANDS_COMPLETE, 0))
 			# Broken in pyglet 2; glFenceSync is missing
 			pass
 		else:
 			gl.glFinish()
 
-	# input functions
-
 	def on_resize(self, width, height):
+		super().on_resize(width, height)
 		logging.info(f"Resize {width} * {height}")
 		gl.glViewport(0, 0, width, height)
 
 		self.player.view_width = width
 		self.player.view_height = height
-		self.f3.y = self.height - 10
-		self.f3.width = self.width // 3
+
+
+class MenuScene(Scene):
+	def __init__(self, window: Window) -> None:
+		super().__init__(window)
+		self.texture_manager = texture_manager.TextureManager(0, 0, 0)
+
+		self.icon = self.texture_manager.load_texture("dirt")
+		self.logo = self.texture_manager.load_texture("logo")
+
+	def on_draw(self):
+		super().on_draw()
+		gl.glEnable(gl.GL_TEXTURE_2D)
+		self.window.clear()
+
+	def update_ui(self):
+		super().update_ui()
+		io = imgui.get_io()
+		imgui.set_next_window_size(io.display_size.x, io.display_size.y)
+		imgui.set_next_window_position(0, 0)
+		imgui.push_style_var(imgui.STYLE_WINDOW_BORDERSIZE, 0.0)
+		imgui.push_style_var(imgui.STYLE_WINDOW_PADDING, (0.0, 0.0))
+
+		if imgui.begin(
+			"MC PY",
```

Comment:
This goes for everywhere MCPY is referred to, but "MCPY" without a space has already been adopted, so it would be nice to stick with that :)

Reply:
Resolved in [de26b9d59a9604287380ab74ed1d0156f4a81dcd](https://github.com/obiwac/python-minecraft-clone/pull/104/commits/de26b9d59a9604287380ab74ed1d0156f4a81dcd).